components/CollectivePickerAsync.js

```diff
@@ -106,18 +106,18 @@ const getPlaceholder = (intl, types) => {
  */
 const CollectivePickerAsync = ({
   inputId,
-  types,
-  limit,
-  hostCollectiveIds,
-  parentCollectiveIds,
-  preload,
-  filterResults,
-  searchQuery,
-  invitable,
-  emptyCustomOptions,
-  noCache,
-  isLoading,
-  skipGuests,
+  types = undefined,
+  limit = 20,
+  hostCollectiveIds = undefined,
+  parentCollectiveIds = undefined,
+  preload = false,
+  filterResults = undefined,
+  searchQuery = collectivePickerSearchQuery,
+  invitable = false,
+  emptyCustomOptions = [],
+  noCache = false,
+  isLoading = false,
+  skipGuests = true,
```

Comment:
Here typescript needed a way to understand that these props are optional. I did not want to convert the entire thing to TS, so I just moved the `defaultProps` to default values, which is anyway the [recommended](https://twitter.com/dan_abramov/status/1133878326358171650) way to do things.